articles/how-to-optimize-wordpress-website-speed/index.md

```diff
@@ -0,0 +1,214 @@
+#### WebP vs PNG
+WebP file size is 26% is smaller than png with the same quality.
+
+Due to the smaller file sizes, WebP loads faster than png.
+#### WebP vs JPEG
+WebP file size is 25% to 35% smaller with the same quality.
+
+Not all browsers can load WebP images. Some browsers will work with jpeg format.
+
+Reducing image size to reduce file is also necessary on
+top of choosing file formats. The main objective is to find a balance
+between file size and quality. Get the lowest file size and acceptable
+quality. Although there exists WordPress image compression plugins,
+compressing images before the upload is recommended. Photoshop,
+[compressor.io](https://compressor.io/) and [BulkResize](https://bulkresizephotos.com/en) are some of the tools used
+for image compression. Image compression plugins include [Imagify](https://imagify.io/wordpress/),
+[ShortPixel](https://wordpress.org/plugins/shortpixel-image-optimiser/),
+[Optimole](https://wordpress.org/plugins/optimole-wp/), and [WP
+Smush](https://wordpress.org/plugins/wp-smushit/). Keep your image under
+100 KB for the best performance.
+
+### Delete Unused Themes, Plugins, and Media Files
+
+This is a crucial website maintenance task. You should get rid of themes
+and plugins that are not in use. The plugins and themes take space in
+the server causing unnecessary strain. This involves deleting installed themes that are not in use and deleting deactivated plugins.
+
+### Use Well-optimized Themes
+
+Don't select some beautiful and impressive looking WordPress themes at
+the expense of performance. Pay special attention to theme optimization. Your
+WordPress theme should be elegant, lightweight, SEO friendly, fully
+responsive, and mobile optimized.
+
+### Optimize WordPress Database
+
+Over time, the WordPress database will accumulate unnecessary data,
+clutter. This will slow down your website. The clutter could be post
+revisions, data that was created by already uninstalled plugins, and
+WordPress sessions. To optimize the database, you can use plugins. It is
+advisable to back up your database before trying to clean. Some of the
+plugins available for database optimization are
+[WP-Optimize](https://wordpress.org/plugins/wp-optimize/) and
+[WP-Sweep](https://wordpress.org/plugins/wp-sweep/).
+
+![](wp-optimize-database-optimization.jpeg)
+
+See above screenshot of WP-Optimize 3.1.2 free version database
+optimization
+
+### Use Lazy Loading for Images and Videos
+
+Lazy loading is a web optimization technique whereby, visible content is
+loaded first while contents appearing down the fold download and
+rendering is delayed. Instead of loading all images and videos, Lazy
+loading downloads and render only what is visible to the user's screen.
+As the user scrolls down the page, more content is loaded. This
+significantly increases the loading speed of the content that is visible
+to the user.
```

Comment:
You could mention how to activate lazy-loading images such as the built-in functionality in the latest WP update.